api/src/main/java/com/mybooks/api/model/Book.java

```diff
@@ -0,0 +1,29 @@
+package com.mybooks.api.model;
+
+import com.sun.istack.NotNull;
+import lombok.*;
+import org.hibernate.annotations.GenericGenerator;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Entity
+public class Book {
+    @Id
+    @NotNull
+    @GeneratedValue(generator = "system-uuid")
+    @GenericGenerator(name = "system-uuid", strategy = "uuid")
+    private String id;
+
+    @NotNull
+    private String title;
+
+    @NotNull
+    private String authorId;
```

Comment:
If the book has only one author then it should be @onetoone relation.
https://www.baeldung.com/jpa-one-to-one
```
@OneToOne
private Author author;
```

If the book has more than one author, then you need @OneToMany relation :
```
@OneToMany
private List<Author> authors
```